berserk/types/opening_explorer.py

```diff
@@ -79,3 +141,35 @@ class OpeningStatistic(TypedDict):
     recentGames: List[Game]
     # top rating games with this opening
     topGames: List[Game]
+
+
+class PlayerOpeningStatistic(TypedDict):
+    # Number of game won by white from this position
+    white: int
+    # Number of game won by black from this position
+    draws: int
+    # Number draws from this position
+    black: int
+    # Opening info of this position
+    opening: Opening | None
+    # The list of moves played by the player from this position
+    moves: List[PlayerMove]
+    # recent games with this opening
+    recentGames: List[Game]
+    # Queue position for indexing (present when wait_for_indexing parameter used)
+    queuePosition: NotRequired[int]
+
+
+class MastersOpeningStatistic(TypedDict):
+    # Number of game won by white from this position
```

Comment:
can be refactored with generic typeddict https://alexocallaghan.com/python-typeddict-with-generics